.github/workflows/npm.yml

```diff
@@ -2,6 +2,12 @@ name: Publish to NPM
 on:
   release:
     types: [published]
+
+permissions:
+  contents: read
+  packages: write
```

Comment:
```suggestion
```
Not required as we don't use GitHub Packages 👍

Reply:
removed.